### ISSUE TITLE
Relative prioritization of annotation features for developers

### ISSUE THREAD

**Issue by mejackreed** [ProjectMirador/mirador-annotations#10]
- [ ] Figure out how to handle clicks on added geometry
 - [ ] Add rotation (for only certain things)
 - [ ] Add resize (for only certain things)
 - [ ] Determine if any additional text editor functionality is desired

**Comment by damooo** [comment]
Hello @mejackreed, Namaste.
We are using mirador2 annotation CRUD ops for transcribing, and also as front-end to identify and classify layout elements, proofreading ocrs. And there are minimal inconveniences which irritates volunteers who fill data. issues are like follows:

1. Cannot alter zoom or pan while drawing shapes. this is highly inconvenient for reasons obvious.  It is not possible to alter this behaviour from outside as state-machine is highly coupled.
2. In cases, when ask volunteers to identify and draw lines in a manuscript quickly, after creation of one anno, the shape selection is getting disabled automatically. And we have to toggle it back for every new anno.
3. It will be good, if we can plug in custom form for annotation-text-editors. It is useful for layout classifications etc. like when user have to draw a rectangle and declare it is an HOCR-Paragraph/HOCR-Line, we can better show them a pre-created selector form.
4. It will be good, if M fires events when user clicked on anno-representation, so that we can plug external tools.

We created a work flow for transcribing, proofreading, layout editing, etc. It uses mirador-annos functionality when ever appropriate. If allowed i can give a demo of that, and specifically show above challenges in practical workflow.

Thank you.

**Comment by mejackreed** [comment]
Hi @damooo thank you for your comment. Yes I think a demo of how you are using annotations would be great to see. Could we setup a meeting? Or do you have a video?

**Comment by damooo** [comment]
@mejackreed  yes, a meeting will be good. through zoom/jitsi/..? can you please message me on slack the medium comfortable, and url? slack-id: damodara
